api4/team.go

```diff
@@ -168,6 +169,26 @@ func getTeamMembers(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func getTeamUnread(c *Context, w http.ResponseWriter, r *http.Request) {
+	c.RequireTeamId()
+	if c.Err != nil {
+		return
+	}
+
+	if !app.SessionHasPermissionTo(c.Session, model.PERMISSION_MANAGE_SYSTEM) {
+		c.SetPermissionError(model.PERMISSION_MANAGE_SYSTEM)
+		return
+	}
```

Comment:
Actually this permission check is incorrect, please check if the user can view the team with `app.SessionHasPermissionToTeam(c.Session, model.PERMISSION_VIEW_TEAM)`